portfolio/src/main/java/com/google/sps/servlets/StudyAbroadServlet.java

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.sps.data.Country;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Returns Country data as a JSON array, e.g. [{"lat": 38.4404675, "lng": -122.7144313}] */
+@WebServlet("/study-abroad")
+public class StudyAbroadServlet extends HttpServlet {
+
+  private Collection<Country> studyAbroad;
+
+  @Override
+  public void init() {
+    studyAbroad = new ArrayList<>();
+
+    Scanner scanner = new Scanner(getServletContext().getResourceAsStream("/WEB-INF/studyAbroadCountries.csv"));
```

Comment:
Add a comments for what the scanner and parser is doing. Or split out the parsing part to a different method.